### PR TITLE
option for customizing test nodename with variable

### PR DESCRIPTION
**Pull request by u3s**

### CODE REVIEW

File: make/test_target_script.sh [hunk 1]
```diff
@@ -291,6 +291,12 @@ then
     exit 1
 fi
 
+if [ -z "${CUSTOM_NODENAME}" ]; then
+  NODENAME=test_server
+else
+  NODENAME="${CUSTOM_NODENAME}"
+fi
+
```
Comment (garazdawi) [inline make/test_target_script.sh:299]:
```suggestion
CT_NODENAME=${CT_NODENAME:-test_server}

```

File: make/test_target_script.sh [hunk 2]
```diff
@@ -311,7 +317,7 @@ then
         -pz "."\
         -ct_test_vars "{net_dir,\"\"}"\
         -noshell\
-        -sname test_server\
+        -sname ${NODENAME}\
```
Comment (garazdawi) [inline make/test_target_script.sh:320]:
```suggestion
        -sname ${CT_NODENAME}\
```

File: make/test_target_script.sh [hunk 3]
```diff
@@ -332,7 +338,7 @@ else
...
```
Comment (garazdawi) [inline make/test_target_script.sh:341]:
```suggestion
        -sname ${CT_NODENAME}\
```

**Comment by u3s** [comment]
I'll squash commits later ...

**Review by garazdawi — APPROVED** [review]

**Pull request merged**